dev.mk

```diff
@@ -0,0 +1,72 @@
+DOCKER_REPO?=quay.io/codeready-toolchain
```

Comment:
why do we have `dev.mk` and `make/dev.mk` ?

Reply:
Oh... good catch. I will delete dev.mk. Not sure what it’s doing here :)

Reply:
Done in https://github.com/codeready-toolchain/registration-service/pull/36/commits/f2684947bcfaa9e2c8dc0b13fc20bc20ff913363